setup.py

```diff
@@ -51,5 +51,5 @@ def run_tests(self):
     cmdclass={'test': PyTest},
     packages=['vulture'],
     package_dir={'vulture': 'vulture'},
-    package_data={'vulture': ['whitelists/*.py']},
+    package_data={'vulture': ['lines.py', 'whitelists/*.py']},
```

Comment:
I don't think we need to mention lines.py at all. We also don't list core.py. `packages=['vulture']` should take care of this already.